dts/arm/nordic/nrf51822.dtsi

```diff
@@ -154,6 +154,7 @@
 			reg = <0x4000b000 0x1000>;
 			interrupts = <11 1>;
 			status = "ok";
+			prescaler = <0>;
```

Comment:
btw, could we have "ranges" in DTS values as well? so, if anyone overwrites with an out-of-range value, DTC gives an error...

Reply:
no sure it DTS supports it as prescaler is generic for all vendors. i can add static assert to source file.

Reply:
thanks

Reply:
static asserts added to timer and rtc drivers.